packages/client/components/bottom-drawer/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+import { useSwipeable } from 'react-swipeable'
+
+const BottomDrawerContainer = styled(Grid)`
+  height: ${props => props.drawerHeight};
+  background-color: white;
+  border-radius: 20px 20px 0px 0px;
+  padding-top: 6px;
+  padding-left: 16px;
+  padding-right: 16px;
+  box-sizing: border-box;
+  transition: height 0.3s;
```

Comment:
Usually, you don't mess around with the flexbox, but rather wrap a component around the flexbox. Adding css directly into the flexbox may make it buggy.